Sources/MapboxMaps/Location/Puck/Puck2D.swift

```diff
@@ -174,11 +174,80 @@ private extension Puck2DConfiguration {
         topImage ?? UIImage(named: "location-dot-inner", in: .mapboxMaps, compatibleWith: nil)!
     }
 
-    var resolvedBearingImage: UIImage {
-        bearingImage ?? UIImage(named: "location-dot-outer", in: .mapboxMaps, compatibleWith: nil)!
+    var resolvedShadowImage: UIImage {
+        shadowImage ?? UIImage(named: "location-dot-outer", in: .mapboxMaps, compatibleWith: nil)!
     }
 
     var resolvedScale: Value<Double> {
         scale ?? .constant(1.0)
     }
 }
+
+public extension Puck2DConfiguration {
+    // Create a Puck2DConfiguration instance with or without an arrow bearing image. Default without the arrow bearing image.
+    static func makeDefault(withBearing: Bool = false) -> Puck2DConfiguration {
```

Comment:
```suggestion
    static func makeDefault(showBearing: Bool = false) -> Puck2DConfiguration {
```

With factory methods, the guidelines say not to form phrases. See screenshot in https://github.com/mapbox/mapbox-maps-ios/pull/980#discussion_r784884048